composer.json

```diff
@@ -31,6 +31,9 @@
     "doctrine/orm": "~2.3",
     "phpdocumentor/reflection": "~1.0"
   },
+  "conflict": {
```

Comment:
It could be apply only if this package is a requirement, not a suggestion.

Please see https://github.com/dunglas/php-property-info/pull/2#discussion_r35088803

ping @Seldaek for opinion.